package.json

```diff
@@ -37,7 +37,7 @@
     "build:widget": "nx build @novu/widget",
     "build:embed": "nx build @novu/embed",
     "build:storybook": "nx run @novu/web:build-storybook",
-    "test:providers": "find providers -name package.json -not -path \"*/node_modules/*\" -exec bash -c \"pnpm --prefix \\$(dirname {}) test\" \\;",
+    "test:providers": "pnpm --filter '{providers}' test",
```

Comment:
the previous approach silently muted error codes from test, those the suite was always passing successfully 😅